snooty/test_semantic_parser.py

```diff
@@ -44,7 +45,9 @@ def test_expand_includes(backend: Backend) -> None:
         {
             "type": "target",
             "position": {"start": {"line": 1}},
-            "ids": ["connection-limits"],
+            "domain": "std",
+            "name": "label",
```

Comment:
It looks like the `name` field is always being output as `label` (in these tests and after running the parser on my machine). Is this expected?

Reply:
That'll be true for any targets defined using the `.. _foo:` syntax